wg-advisors/red-flags.md

```diff
@@ -15,7 +15,7 @@ not a policeman enforcing the rules.
 * Has a website external to apache.org
 * Is missing reports
 * Is missing release votes
-* Has too many release candidates
+* Has too many release candidates: [why](why/why_release_process.md)
```

Comment:
The answer is not included in the new file. Do you expect elaborate it in the new file later, or we should create a dedicated answer.

I remember that the original purpose of this "red flag" was to encourage RMs to waiting for more inputs instead of eagerly cancelling release candidates which increases the release burden.

Reply:
It's an interesting one. This is precisely the kind of discussions that I **hoped** the `why` document will start :)

IMHO that one actually deserves a separate "why", as this red flag is (now that you explained it not really about software releases "per se" but about communication and process around the releases, and possibly about involving the community during the relase process and engaging them in testing  - whcih is a subtly different thing and possibly deserves another "why". I wil propose something soon :)

Reply:
I tried to separate it, but indeed it seems better if it is combined in the release process. I just stressed how important it is to engage the dev community in the release/testing process and added a little description summarising why we are doing it. Should be explained now.